index.js

```diff
@@ -12,37 +12,459 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+    anchor2.classList.add('ruby');
+    anchor2.textContent = 'Ruby on rails';
+  }
+  btn2.appendChild(anchor2);
+  // create li
+  const btn3 = document.createElement('li');
+  if (projects[i].name === 'Tonic') {
+    btn3.classList.add('tag3');
+  } else if (projects[i].name === 'Multi-Post Stories' && window.innerWidth < 768) {
+    btn3.classList.add('tag3');
+  } else if (projects[i].name === 'Multi-Post Stories' && window.innerWidth > 768) {
+    btn3.classList.add('tag2');
+  } else if (projects[i].name === 'Facebook 360') {
+    btn3.classList.add('tag2');
+  } else if (projects[i].name === 'Uber Navigation') {
+    btn3.classList.add('tag2');
+  }
+  ul.appendChild(btn3);
+  // create anchor
+  const anchor3 = document.createElement('a');
+  if (projects[i].name === 'Tonic') {
+    anchor3.classList.add('js');
+    anchor3.textContent = 'javascript';
+  } else if (projects[i].name === 'Multi-Post Stories' && window.innerWidth < 768) {
+    anchor3.classList.add('js');
+    anchor3.textContent = 'javascript';
+  } else if (projects[i].name === 'Multi-Post Stories' && window.innerWidth > 768) {
+    anchor3.classList.add('css');
+    anchor3.textContent = 'css';
+  } else if (projects[i].name === 'Facebook 360') {
+    anchor3.classList.add('css');
+    anchor3.textContent = 'css';
+  } else if (projects[i].name === 'Uber Navigation') {
+    anchor3.classList.add('css');
+    anchor3.textContent = 'css';
+  }
+  btn3.appendChild(anchor3);
+  // create li
+  const btn4 = document.createElement('li');
+  if (projects[i].name === 'Multi-Post Stories' && window.innerWidth > 768) {
+    btn4.classList.add('tag3');
+  } else if (projects[i].name === 'Facebook 360') {
+    btn4.classList.add('tag3');
+  } else if (projects[i].name === 'Uber Navigation') {
+    btn4.classList.add('tag3');
+  }
+  ul.appendChild(btn4);
+  // create anchor
+  const anchor13 = document.createElement('a');
+  if (projects[i].name === 'Multi-Post Stories' && window.innerWidth > 768) {
+    anchor13.classList.add('js');
+    anchor13.textContent = 'javascript';
+  } else if (projects[i].name === 'Facebook 360') {
+    anchor13.classList.add('js');
+    anchor13.textContent = 'javascript';
+  } else if (projects[i].name === 'Uber Navigation') {
+    anchor13.classList.add('js');
+    anchor13.textContent = 'javascript';
+  }
+  btn4.appendChild(anchor13);
+  // create button
+  const proj = document.createElement('button');
+  if (window.innerWidth < 768) {
+    proj.classList.add('see-project');
+    proj.textContent = 'see project';
+  } else if (window.innerWidth > 768) {
+    proj.classList.add('project-desktop');
+    proj.textContent = 'see project';
+  }
+  cardBody.appendChild(proj);
+
+  projectsContainer.appendChild(card);
 }
+window.addEventListener('resize', () => this.location.reload());
+
+// modal
+const see = document.querySelectorAll('.see-project');
+see.forEach((btn) => (btn.addEventListener('click', () => {
```

Comment:
**I can see you put great efford on this project.** 
_However, we have one thing left to do!_

- [ ] As the requirement your popup window info should relete to the project that we are opening and the data should be gotten from the array you created. We can see in your popup window only one data. Kindly manipulate the popup window data based on array and should matcha the project that you opened. Thank you!